docs/source/plugins/for_plugin_developers.rst

```diff
@@ -6,41 +6,47 @@
 creating a napari plugin
 ========================
 
-This document explains how to extend napari functionality by writing a plugin
+This document explains how to extend napari's functionality by writing a plugin
 that can be installed with ``pip`` and autodetected by napari.  For more
-information on how plugins are implementated internally in napari, see
+information on how plugins are implemented internally in napari, see
 :ref:`plugins-for-napari-developers`.
 
 
 Overview
 --------
 
-``napari`` allows developers to extend the functionality of the program as
-follows:
-
-1. **Hooks**: We (napari) identify specific places ("*hooks*") in the napari
+``napari`` allows developers to extend its functionality through **hooks**,
+specific places in the napari
 codebase where we would like external developers to be able to extend
```

Comment:
now that this is part of the same sentence... the repeated usage of "extend functionality" reads funny